### ISSUE TITLE
Feat: Feature toggle for Schema Generation

### ISSUE THREAD

**Issue by roshaans** [near/queryapi#276]
Schema generation is not supported for all possible schemas, so we will add a feature flag for schema generation until this feature is stable.

**Comment by pkudinov** [comment]
Do you mean the schema generation for `context.db` types?

**Comment by roshaans** [comment]
This does not effect all indexers. Only a select few.

e.g https://near.org/dataplatform.near/widget/QueryApi.App?selectedIndexerPath=roshaan.near/bos_widgets&view=editor-window

<img width="1395" alt="image" src="https://github.com/near/queryapi/assets/25015977/f66fce4d-8ebc-4182-b4f1-819f6fc3aaaa">

**Comment by pkudinov** [comment]
I see. Why do we need a feature flag though? We can just notify a user that this schema is not working and let them proceed. We can change red color to amber. This ticket can be used to track the ability to support this type of DDL

**Comment by roshaans** [comment]
good idea. I agree that would be a better way to handle this. So i'll just make the error exitable + remove the check for schema generation when the user tries to publish the indexer. One thing to note is that V2 indexer execution will fail because it also depends on this schema generation capabilities.
